packages-backend/express/src/auth.ts

```diff
@@ -148,9 +154,17 @@ function createSessionAuthVerifier<
       issuer = mapToLegacyIssuer(cloudIdentifierHeader) ?? issuer;
     }
 
-    // @ts-ignore: the node HTTP request does not know about `originalUrl`
-    const requestUrlPath = request.originalUrl ?? request.url;
-    const audience = getConfiguredAudience(options, requestUrlPath);
+    const requestUrlPath = options.getRequestUrl
+      ? options.getRequestUrl(request)
+      : request.originalUrl ?? request.url;
+
+    if (!requestUrlPath) {
+      throw new Error(
+        'Invalid request path. Please make sure that the request object has either a property `originalUrl` or `url`. If not, you should implement the `getRequestUrl` function. More info at https://docs.commercetools.com/custom-applications/concepts/integrate-with-your-own-api#validating-the-json-web-token'
```

Comment:
```suggestion
        'Invalid request URI path. Please make sure that the `request` object has either a property `originalUrl` or `url`. If not, you should implement the `getRequestUrl` function. More info at https://docs.commercetools.com/custom-applications/concepts/integrate-with-your-own-api#validating-the-json-web-token'
```

Reply:
Updated in this commit e4be0ae17a4678626bf426a1c7ab237e458a00d1